backend/src/index.ts

```diff
@@ -25,9 +25,6 @@ import databaseMigration from './api/database-migration';
 import syncAssets from './sync-assets';
 import icons from './api/liquid/icons';
 import { Common } from './api/common';
-import mining from './api/mining';
```

Comment:
@nymkappa did you mean to remove these? It's now showing up as a merge conflict

Reply:
Rebased